tests/functional/u/unspecified_encoding_py38.txt

```diff
@@ -31,3 +31,11 @@ bad-open-mode:161:0:161:25::"""None"" is not a valid mode for open.":UNDEFINED
 unspecified-encoding:161:0:161:25::Using open without explicitly specifying an encoding:UNDEFINED
 bad-open-mode:164:0:164:19::"""2"" is not a valid mode for open.":UNDEFINED
 unspecified-encoding:164:0:164:19::Using open without explicitly specifying an encoding:UNDEFINED
+bad-open-mode:176:0:176:24::"""5"" is not a valid mode for open.":UNDEFINED
```

Comment:
Here the confidence level should be INFERENCE for all the new case, and be HIGH for the existing cases. It means we're going to have to get that information from ``get_argument_from_call``. Maybe it's an opportunity to not modify ``get_argument_from_call`` signature and do things differently because changing the return would complexify the design too much ? We might want to add the confidence HIGH in another MR first if it's harder than simply adding the parameter to ``add_message`` (I did not check).

Reply:
Interesting, I will research into adding CONFIDENCE levels and get back

Reply:
See this snippet:

https://github.com/pylint-dev/pylint/blob/6bfa0402fe1aaf3db8b40b30afa979eb0156aa84/pylint/checkers/bad_chained_comparison.py#L51-L56

Reply:
Thanks, I have added the confidence levels, I had to change the bad_mode.txt as well since now it outputs HIGH confidence as well.